Shokofin/Sync/UserDataSyncManager.cs

```diff
@@ -233,6 +238,11 @@ public async void OnUserDataSaved(object sender, UserDataSaveEventArgs e)
                     }
                 }
             }
+            catch (HttpRequestException ex) when (ex.StatusCode is HttpStatusCode.Unauthorized) {
+                TryGetUserConfiguration(e.UserId, out var userConfig);
+                if (userConfig is not null)
```

Comment:
The `TryGet*` pattern allows us to get a variable while at the same time return a boolean to indicate the success or failure to retrieve the value.  While it can be used on a separate line, that's usually when you don't care if the value is successfully retrieved or not... but here we care, so we can just use it in the `if` statement to check if the value is not `null`.
```suggestion
                if (TryGetUserConfiguration(e.UserId, out var userConfig))
```